Use new provisioning approach (#4)
mrc-ide/wodin-epimodels: README.md

* `config/msc-idm-2022`: the 2022 MSc course
* `config/malawi-idm-2022`: a short course run in Malawi
* `config/gambia-idm-2023`: a short course run in The Gambia

## Deploying

```
git clone git@github.com:mrc-ide/wodin-epimodels.git
```

Configure the proxy by running `./configure-proxy`:

* `./configure-proxy epimodels` for use on epimodels.dide
* `./configure-proxy wodin-dev` on wodin-dev.dide
* `./configure-proxy localhost` locally for testing (no ssl)

This will copy any required ssl keys to `ssl/` and write out the required hostname into the file `hostname`.

Then deploy (or redeploy) with `./deploy`

## Adding a new site

Edit [`deploy`](deploy) to:

* list the new site within the array variable `SITES` (first line in the file basically)
* add an entry in `SITES_URL` just below that; the key (in square brackets) must match the new entry in `SITES` exactly
* if you need a specific reference add that as `SITES_REF[newsite]=mrc-1234`, otherwise this defaults to `main`

For example adding

```
SITES=(... newsite)
SITES_URL[newsite]=https://github.com/mrc-ide/wodin-newsite
SITES_REF[newsite]=prototype
```

> would make a new site available at `http://localhost/newsite` using configuration from `https://github.com/mrc-ide/wodin-newsite` using branch `prototype`. Delete this last line once things have settled.

Reviewer: ```suggestion
would make a new site available at `http://localhost/newsite` using configuration from `https://github.com/mrc-ide/wodin-newsite` using branch `prototype`. Delete this last line to start using the `main` branch.
```
